src/main/java/frc/robot/DataLogging.java

```diff
@@ -137,6 +140,10 @@ public void periodic() {
     if (Constants.LOOP_TIMING_LOG) {
       loopTime.append(Timer.getFPGATimestamp() - startTime);
     }
+
```

Comment:
The Contants.LOOP_TIMING_LOG clause needs to be the last thing in the periodic method.

Reply:
Done